src/libs/ajax/Groups.ts

```diff
@@ -84,6 +84,12 @@ export const Groups = (signal?: AbortSignal) => ({
         const res = await fetchSam(`${resourceRoot}/action/use`, _.merge(authOpts(), { signal }));
         return res.json();
       },
+
+      // we could provide a more specific type here, but we don't use it and don't want to limit future additions
```

Comment:
Since it looks like state that stores this response is also typed `object`, suggest defining an alias like `type SupportSummary = object` and using that. It's equivalent to `object` in terms of type checking, but it serves as documentation for what the state variables store.